.circleci/windows/erlang_install.bat

```diff
@@ -0,0 +1,20 @@
+@echo off
+IF NOT "x%1"=="x" IF NOT "x%2"=="x" GOTO:INSTALL
+@echo Usage: %0 ^<otp_version^> ^<install_path^>
+exit /b 2
+
+:INSTALL
+@echo %time% : Install Erlang/OTP %1
+IF EXIST %2 GOTO:INSTALLED
+
+SET "URL=https://packages.erlang-solutions.com/erlang/erlang/esl-erlang/FLAVOUR_1_general/esl-erlang_%1~windows_amd64.exe"
+@echo %time% : Download %URL% started
+curl -s -o "%temp%\erl%1.exe" "%URL%"
+IF NOT "%ERRORLEVEL%"=="0" (echo %time% : Download failed && exit /b %ERRORLEVEL%)
+
+@echo %time% : Install started
+powershell start-process -wait '%temp%\erl%1.exe' -argumentlist '/S /D=%2'
```

Comment:
since the installer always runs as a separate process we need to hold until it finish